pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>2.150.1</jenkins.version>
+        <java.level>8</java.level>
```

Comment:
Well to keep my head sane, I'll focus on java 8 and Jenkins 2.60.3 at the minimum but I properly would aim higher 😅

Reply:
Well, if you want to do the work required to bring this plugin into the Jenkins-2 world that then that's OK with me, but I think it would be better done as a separate PR.
Personally, I would recommend bumping the Jenkins version to a fairly early Jenkins-2 (e.g. 2.60.3) because there might be folks "out there" using this plugin that are still using early versions.
e.g. where I work, we typically upgrade Jenkins once or twice a year (unless there's serious security concerns) - it already does what we need it to do, so there's little benefit to upgrading.

FYI If the files changed by doing that and by the JCasc changes don't overlap then it should be trivial to work on both PRs in parallel (which we'll only need to do until the Java8+Jenkins2.60.3 PR is merged in, which should only be a matter of days at most) and then the JCasc changes should auto-merge without additional human intervention.

Reply:
Being done in a separate PR